CMakeLists.txt

```diff
@@ -307,7 +312,9 @@ set_target_properties(lib${APP_NAME_LC} PROPERTIES PREFIX "")
 add_dependencies(lib${APP_NAME_LC} libcpluff ffmpeg dvdnav crossguid)
 
 # Codegen
-add_subdirectory(${CMAKE_SOURCE_DIR}/xbmc/interfaces/swig build/swig)
+if(ENABLE_PYTHON)
```

Comment:
sure i can, but i thought you had it explicitly in the buildsystem for a reason (other than technical).